synth.py

```diff
@@ -31,41 +31,10 @@
     s.copy(library / 'system-test')
     s.copy(library / 'test')
 
-'''
-Build private versions
-'''
-gapic = gcp.GAPICGenerator(private=True)
-
-versions = []
-
-for version in versions:
-    library = gapic.node_library('texttospeech', version)
-
-    s.copy(library / 'protos')
-    s.copy(library / 'src' / version)
-    s.copy(library / 'samples')
-    s.copy(library / 'system-test')
-    s.copy(library / 'test')
 
 '''
 Node.js specific cleanup
 '''
-# Repo Cleanup/Setup
```

Comment:
Can we just use `s.copy(library, excludes=...)` style here? Instead of calling out each dir?